install_config/aggregate_logging.adoc

```diff
@@ -836,11 +836,11 @@ $ for dc in $(oc get deploymentconfig --selector logging-infra=elasticsearch -o
 ----
 
 . The Elasticsearch replicas must be located on the correct nodes to use the local
-storage, and should not move around even if those nodes are taken down for a
+storage, and should not move around, even if those nodes are taken down for a
```

Comment:
s/storage,/storage
Please rewrite to remove "should."

I'd also try to put the action earlier in the step.

Reply:
Thanks - deleted comma after storage and reworded to remove "should."
Tried moving action earlier, but there are enough explanatory remarks about replicas, it might be more awkward and confusing to do so.